docs/csharp/csharp-7.md

```diff
@@ -432,53 +426,59 @@ work begins:
 > could also be accomplished using *lambda expressions*. Those
```

Comment:
another existing typos: minimimum (line 118), declaraed (line 377) 
mis-use should have no hyphen (lines 292 and 350)
left hand and right hand should be hyphened (line 103, 108, 335)